documentation/sphinx/source/release-notes/release-notes-7131.rst

```diff
@@ -0,0 +1,198 @@
+.. _release-notes:
+
+#############
+Release Notes
+#############
+
+71.3.1
+======
+
+Features
+--------
+- Add HoltLinearSmoother class
+- EKP and KMS Health Check
+- Adding blob granule restarting tests to use tenants, encryption
+- Implement IStorageMetricsService interface
+- Make stepSize configurable for preloadApplyMutationsKeyVersionMap
+- Add networkoption to disable non-TLS connections
+- New ConsistencyScan
+- Add WatchMapSize and OutstandingWatches to TransactionMetrics 
+- Add blob manifest url and mlogs url in status json. 
+- Add tenant_id_prefix to metacluster status
+- Support blob granule restore in fdbrestore 
+- Move lastFlushTs to BlobGranuleBackupConfig 
+- Cache change feeds durably on blob workers
+- Support restoring a cluster with a tenant in the error state
+- EaR RESTSimKMS Vault 
+- EaR: Update KMS URL refresh policy and fix bugs 
+- EaR: Add test case to validate decryption with invalid key
+- Decouple token bucket knobs for different types of throttlers 
+- Adding blob ranges to backup keys 
+- Allow reading from system keyspace even with default tenant
+- Add ExpectStableThroughput simulation test 
+- Simplify GlobalTagThrottler limiting TPS calculation 
+
+
+Performance
+-----------
+- Re-enabling change feed coalesce knob
+- Tune default GlobalTagThrottler knobs
+- Improve performance of TransactionTagCounter
+- Update knob values for Storage Quota polling intervals
+- Removing verbose logs that are not useful for 71.3
+- Remove some unnecessary ref-counting in the PTree
+- Adding delta file count before resnapshot to minimize impact of small delta files   
+- EaR: reduce metrics logging 
+- Made blob metadata load lazily from EKP
+- Fix retransmits in corruption check
+- Increase coverage for slow tests in TestHarness2
+- Dump manifest by using multiple transactions 
+- EaR: Optimize logging from GetEncryptCipherKey 
+- Speed up BackupCorrectnessClean
+- Speed up DR upgrade tests
+
+
+Reliability
+-----------
+- Fixing stable feeds check by allowing more time post speedup sim
+- Check if var is None before comparing with int
+- Handle operation_cancelled properly in configuration database
+- Reset connection idle time when restarting connection monitor
+- BlobGranuleRestore - skip muations applying if restore target version
+- Update SimKmsVault unit test assert checking max encryption keys
+- Check serverList before update storage metadata 
+- Test use of the metacluster after a restore 
+- Decrease the number of records in MutationLogReaderCorrectness 
+- EaR: Remove usage of ENABLE_CONFIGURABLE_ENCRYPTION knob
+- Only starting bg restarting correctness
+- Ignore AutomaticIdempotency tests 
+- Remove mockkms dir from the build dir
+
```

Comment:
Probably makes sense to keep the bullet formatting consistent between the first and second half

Reply:
Observing past release notes, `Fixes` section bullet usage has been `*` and other section it has been `-`, I don't have any strong opinion, so followed the same for consistency reasons.